package.json

```diff
@@ -76,14 +76,14 @@
     "delay": "^4.3.0",
     "dirty-chai": "^2.0.1",
     "it-pair": "^1.0.0",
+    "libp2p": "libp2p/js-libp2p#chore/deprecate-old-peer-store-api",
```

Comment:
@jacobheun I am not sure if mocking the `peerStore` is the best thing to do here, or depend on `libp2p` as a dev dependency again. What do you think? It could be worth having a mocks repo for this kind of stuff, but we would need to make sure it was properly updated with new changes

Reply:
yeap, seems good!